server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -73,6 +96,10 @@ object NettyConfig {
     shutdownEventLoopGroupOnClose = true,
     socketBacklog = 128,
     socketKeepAlive = true,
+    requestTimeout = None,
```

Comment:
don't we want to have some kind of defaults? what are the defaults in other http servers?

Reply:
I guess this would just leave Netty's default behavior. It's different than other timeouts though, so maybe we should set a value here.
Looking at other servers (https://github.com/http4s/http4s/issues/2550#issuecomment-513568950) we could use a timeout of sth like 20, 30 seconds?

Reply:
I think I'd set some values, even for the sake of documenting Netty's defaults. But even better would be looking at akka-http, http4s for sane default values and using them here.